.github/workflows/image-pr.yaml

```diff
@@ -187,6 +187,20 @@ jobs:
   #  needs:
   #    - core-alpine
 
+  custom-partitioning:
+    uses: ./.github/workflows/reusable-custom-partitioning-test.yaml
+    with:
+      flavor: ${{ matrix.flavor }}
+      flavor_release: ${{ matrix.flavorRelease }}
+    needs:
+      - core
+    strategy:
+      fail-fast: true
+      matrix:
+        flavor:
+          - "ubuntu"
+        flavorRelease:
+          - "23.10"
```

Comment:
if this gets merged after https://github.com/kairos-io/kairos/pull/2447, we should add 24.04 here, or if it's the other way around then we should add it in the other PR, putting a comment there too